api/api_sources/sources/libs/transformer/db.transformer.ts

```diff
@@ -76,7 +76,7 @@ export class DateTimeTransformer {
             return date;
         }
         try {
-            const ds = `${moment(date).format('YYYY-MM-DD hh:mm:ss')}`;
+            const ds = `${moment(date).format('YYYY-MM-DD HH:mm:ss')}`;
```

Comment:
@fwpushan I had to change the format here as it does not provide the `am/pm` or the 24hrs part of the time. Personally, I feel that it would be better to send the timestamp as how it is stored in the db and the frontend could format it in the way we want to display. Unless there is a reason why you added it here

Reply:
This is a problem with ORM which basically sometime returns date/timestamp field as TS/JS Date object. And iOS app also expects data in that format.